codes/classical/bits/random_code.yml

```diff
@@ -0,0 +1,128 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#} (PREFERRED),
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: random_code
+physical: bits
+logical: bits
+
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title (SINGULAR) + first reference(s) (optional).
+name: 'Random code'
+introduced: '\cite{doi:10.1109/TIT.2002.800480}'
+
+# Anything applicable to a larger parent set of codes (see below) should go in
+# that entry instead of here.
+description: 'A random binary code is a set of 2^{K}=2^{NR} codewords with length N.Each bit in the codeword is randomly chosen between 0 and 1 with equal probability.
+              A random binary linear code is generated from a random chosen K generators of length N where each bit of the generators is randomly chosen between 0 and 1 with equal probability.
+              Equivalently, A random binary linear code is defined by a randomly generated K by N generator matrix where each entry is randomly chosen between 0 and 1 with equal probability.'
+
+
+
+
+# Long fields such as this one can be written in other YML formats, such as the one using the pipe symbol
+# protection: |
+#   text...
+#   more text...
+protection: 'The code can protect against random errors, e.g. from Binary Symmetry Channel (BSC), discrete memoryless channel.'
+
+
+
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features: 'Given N and the rate R, the minimum distance of a typical random code (TRC) is given by the Gilbert–Varshamov (GV) bound d=N\delta_{GV}(2R).
```

Comment:
We need to define the function \delta_{GV}, namely \delta_{GV}(x):= h^{-1}(1-x), 0\le x \le 1,  and \delta_{GV}(x)=0 if for all other x.